### PR TITLE
CRUD STAFF OPERATION Go to Staff Page from Course Page

### LINKED ISSUE #37: Staff Crud Operations (admin) Go to Staff Page from Course Page
User Story:
I can go to the staff page from the course page

### PR DESCRIPTION
**Pull request by andrew-kwon1**
User Story:
I can go to the staff page from the course page

dokku dev build: https://organic-andrewkwon-dev.dokku-11.cs.ucsb.edu

Clicking on staff button on the course page leads to a placeholder staff page. 
image

image

Other members can now work on other crud operations. There maybe some merge conflicts with App.js
Closes #37

### CODE REVIEW

**Review by yuxiaolejs — DISMISSED** [review]
LGTM

File: frontend/src/main/pages/CoursesStaffPage.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+//import { useParams } from "react-router-dom";
+//import StaffTable from 'main/components/Staff/StaffTable';
+//import { useBackend } from "main/utils/useBackend";
```
Comment (pconrad) [inline frontend/src/main/pages/CoursesStaffPage.js]: Please remove commented out code

File: src/main/resources/application-development.properties [hunk 2]
```diff
@@ -8,7 +8,7 @@ spring.datasource.url=jdbc:h2:file:./target/db-development
 spring.datasource.username=sa
 spring.datasource.password=password
 spring.h2.console.settings.web-allow-others=true
-spring.h2.console.enabled=true
+spring.h2.console.enabled=${H2_CONSOLE_ENABLED:${env.H2_CONSOLE_ENABLED:true}}
```
Comment (pconrad) [inline src/main/resources/application-development.properties]: Why is this in this branch?

File: src/main/resources/application-production.properties [hunk 3]
```diff
@@ -9,4 +9,5 @@ spring.liquibase.user=${JDBC_DATABASE_USERNAME}
 spring.liquibase.password=${JDBC_DATABASE_PASSWORD}
 spring.liquibase.enabled=true
 
-app.showSwaggerUILink=${SHOW_SWAGGER_UI_LINK:${env.SHOW_SWAGGER_UI_LINK:false}}
\ No newline at end of file
+app.showSwaggerUILink=${SHOW_SWAGGER_UI_LINK:${env.SHOW_SWAGGER_UI_LINK:false}}
+spring.h2.console.enabled=${H2_CONSOLE_ENABLED:${env.H2_CONSOLE_ENABLED:false}}
```
Comment (pconrad) [inline src/main/resources/application-production.properties]: This should not be in this branch.

File: src/main/resources/application-development.properties (hunk 2 above)
Comment (pconrad) [reply src/main/resources/application-development.properties]: Please remove this from this branch.

**Review by pconrad — CHANGES_REQUESTED** [review]
Please remove the commented out code, and also fix the branch so it's not tangled up with changes from another PR.

File: frontend/src/main/pages/CoursesStaffPage.js (hunk 1 above)
Comment (andrew-kwon1) [reply frontend/src/main/pages/CoursesStaffPage.js]: Removed

File: src/main/resources/application-development.properties (hunk 2 above)
Comment (andrew-kwon1) [reply src/main/resources/application-development.properties]: Removed

File: src/main/resources/application-production.properties (hunk 3 above)
Comment (andrew-kwon1) [reply src/main/resources/application-production.properties]: Removed

**Review by pconrad — DISMISSED** [review]
LGTM

**Pull request merged**